parm/archive/gdas.yaml.j2

```
@@ -30,6 +30,15 @@ gdas:
         - "logs/{{ cycle_YMDH }}/{{ RUN }}_ocnanalecen.log"
                 {% endif %}
             {% endif %}
+            {% if DO_AERO_ANL %}
+        - "logs/{{ cycle_YMDH }}/{{ RUN }}_aeroanlgenb.log"
```

I don't believe there is a `gfs_aeroanlgenb` job, only a `gdas_aeroanlgenb` job.  I'm OK with grabbing this log for both `gfs` and `gdas` `RUN`s for simplicity
```suggestion
        - "logs/{{ cycle_YMDH }}/{{ RUN }}_aeroanlgenb.log"
```

---

There are two other alternatives:
1. Wrap this file in a `if RUN == gdas` or
2. Place this file in the list of "optional" files (this will raise a warning for `RUN == gfs`)

Perhaps there is another way to do this, but for now I think hardcoding the `RUN` is fine.